lstchain/mc/tests/test_plot_utils.py

```diff
@@ -1,18 +1,18 @@
+import matplotlib.pyplot as plt
 from lstchain.mc import plot_utils
 import astropy.units as u
 import numpy as np
 
 
 def test_plot_Crab_SED():
-
+    plt.figure()
     plot_utils.plot_Crab_SED(1*u.GeV, 1*u.PeV, percentage=45, label=r'Crab')
 
 
 def test_sensitivity_plot_comparison():
-
     energy = np.geomspace(0.01, 100, 20) * u.TeV
     e_center = np.sqrt(energy[:-1] * energy[1:])
-    sensitivity = e_center**-1 * 1e-12 / (u.cm**2 * u.s)
-
+    sensitivity = e_center**-1 * 1e-12 * u.TeV**2/(u.cm**2 * u.s)
+    plt.figure()
```

Comment:
why do you need to call `plt.figure()` twice?

Reply:
why twice? in the two different tests you mean?
Creating a new figure is necessary in unit tests to make sure the plot is not added to an existing one (from a previous test) that has different axis units, leading to an error.

Reply:
damn, the line in which a different test was defined must have been collapsed... sorry!